package.json

```
@@ -17,7 +17,8 @@
     "alex": "alex .",
     "test": "cd packages/react-scripts && node bin/react-scripts.js test",
     "format": "prettier --trailing-comma es5 --single-quote --write 'packages/*/*.js' 'packages/*/!(node_modules)/**/*.js'",
-    "compile:lockfile": "node tasks/compile-lockfile.js"
+    "compile:lockfile": "node tasks/compile-lockfile.js",
+    "publish-package": "echo $GOOGLE_AUTH_B64 | base64 --decode > .google_creds && GOOGLE_APPLICATION_CREDENTIALS=.google_creds npx google-artifactregistry-auth .npmrc && rm -f .google_creds && yarn publish"
```

```suggestion
    "publish-package": "echo $GOOGLE_AUTH_B64 | base64 --decode > .google_creds && GOOGLE_APPLICATION_CREDENTIALS=$(cat .google_creds) npx google-artifactregistry-auth .npmrc && rm -f .google_creds && yarn publish"
```
I'm assuming you want to read the contents of .google_creds and not just set the credentials as ".google_creds"

---

Just an update, I went to double check this with Denys and he said, the docs indicate it needs to be the file path not contents of the file.

For reference:
https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable